modules/agent_role/json/policy.json

```diff
@@ -6,6 +6,33 @@
             "Effect": "Allow",
             "Action": "sts:AssumeRole",
             "Resource": "arn:aws:iam::*:role/${prefix}_CstSecurityInspectorRole"
+        },
+        {
+            "Sid": "Statement1",
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "arn:aws:logs:*:*:*"
+        },
+        {
+            "Sid": "Statement2",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:CreateNetworkInterface",
+                "ec2:DescribeNetworkInterfaces",
+                "ec2:DetachNetworkInterface",
+                "ec2:DeleteNetworkInterface"
```

Comment:
What's the `*NetworkInterface*` bit about?

Reply:
These are the permissions chalice creates when you let it create its own policy. I think it's creating an interface from the lambda into the VPC.

Reply:
Ah okay, makes sense.